cropper/src/main/java/com/canhub/cropper/BitmapUtils.kt

```diff
@@ -106,7 +102,11 @@ internal object BitmapUtils {
             orientationAttributeInt == ExifInterface.ORIENTATION_TRANSPOSE
         val flipVertically = orientationAttributeInt == ExifInterface.ORIENTATION_FLIP_VERTICAL ||
             orientationAttributeInt == ExifInterface.ORIENTATION_TRANSVERSE
-        return RotateBitmapResult(bitmap, degrees, flipHorizontally, flipVertically)
+
+        return when (bitmap) {
+            null -> RotateBitmapResult(null, degrees, flipHorizontally, flipVertically)
+            else -> RotateBitmapResult(rotateAndFlipBitmapInt(bitmap, degrees, flipHorizontally, flipVertically), 0, false, false)
```

Comment:
calling `rotateAndFlipBitmapInt` is the fix and once we've flipped and rotated we no longer need to rotate / flip hence `0, false, false`